### DISCUSSION TITLE
Possible New Exploit?

### DISCUSSION THREAD

**Discussion by FunnyCheese4Me** [3kh0/ext-remover#1097] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Also, make sure to have restore apps enable to always on school profile.

So, I just now found, that if you use the skiovox method to open both your school and skiovox window at the same time, you can open crosh in your school profile in both EVEN if it is blocked on the school profile. But, when you log back in to your school profile(I did vmc create-extra-disk --size 1 /home/chronos/user/Extensions/{extension_id} in the crosh on school profile before logging out of skiovox) it gave me:

Launching vmshell failed: crosh not allowed

Picture:

![Screenshot 2024-05-30 8 56 47 AM](https://github.com/3kh0/ext-remover/assets/170764105/701b55c6-6f8f-4084-98f8-8272e4150eb5)

**Comment by S-PScripts** [comment]
oh

**Reply by FunnyCheese4Me** [reply]
I don't know if this could be used somehow, but it could be useful. Maybe disable extensions with crosh v2?

**Comment by FunnyCheese4Me** [comment]
You can even duplicate the tab to access it more, but it gives the same error. But if you just type it in a new tab, it will give a blocked by admin message.
